.github/ISSUE_TEMPLATE/rule_change.yml

```diff
@@ -0,0 +1,56 @@
+name: Rule change
+description: As rule could be improved or extended?
+title: "`rule-name` should do xyz"
+labels:
+  - enhacement
+  - evaluating
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Before opening a new issue:
+        - Look for existing [open or closed rule proposals](https://github.com/sindresorhus/eslint-plugin-unicorn/issues?q=is:issue+label%25enhancement
+        - [Report a bug](https://github.com/sindresorhus/eslint-plugin-unicorn/issues/new?assignees=&labels=&projects=&template=bug_report.md) instead if it's a false positive
+  - type: textarea
+    validations:
+      required: true
+    attributes:
+      label: Description
+      description: Explain here what change or improvement you'd like.
+  - type: textarea
+    validations:
+      required: true
+    attributes:
+      label: Fail
+      description: Specify examples of code that should be detected
+      value: |
+        ```js
+        var replace = 'me';
+        ```
+
+        ```js
+        function foo() {
+          var replace = 'me';
+          return replace;
+        }
+        ```
+  - type: textarea
+    validations:
+      required: true
+    attributes:
+      label: Pass
+      description: Specify examples of code that would be accepted in its place
+      value: |
+        ```js
+        const replace = 'me';
+        ```
+
+        ```js
+        function foo() {
+          return 'me';
+        }
+        ```
```

Comment:
Is this really necessary for this type of issue?

Reply:
The "good code" isn't needed if the suggestion is:

- _"Also detect this other way to do the same thing"_
	-  like https://github.com/sindresorhus/eslint-plugin-unicorn/issues/2181

but it's definitely needed if it's:

- _"Also autofix this other similar but unrelated method"_
	- Like https://github.com/sindresorhus/eslint-plugin-unicorn/issues/2085